{
    "data_folder_path": "data/CVE-2020-6174",
    "pull_requests_urls": [
        "https://github.com/theupdateframework/tuf/pull/974"
    ],
    "cve_name": "CVE-2020-6174",
    "repo": "theupdateframework/tuf",
    "warnings": {
        "cve": "CVE-2020-6174",
        "warnings": []
    },
    "files_extension_stats": {
        ".py": 3
    },
    "file_number": 3,
    "removed_files_number": 0,
    "added_files_number": 0,
    "nb_lines_affected": 176,
    "private_key_files": [],
    "languages": [
        "Python"
    ]
}